source/install/cloud-changelog.md

```diff
@@ -4,12 +4,47 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 - [Release 2022-03-02](#release-2022-03-02)
 - [Release 2022-02-16](#release-2022-02-16)
-- [Release 2022-02-10](#release-2022-02-10)
+
+## Release 2022-04-13
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a configuration setting ``EmailSettings.EnableInactivityEmail`` to add the ability to toggle sending inactivity email notification to Admins.
+ - Added Files and Pinned Messages to the right-hand side Channel Info.
+ - Updated the New Channel modal.
+ - Added another 'Active' filter for users and Admins in **System Console > User Management > Users**.
+ - Removed cloud user limit count.
```

Comment:
let's not include this one in the changelog (since this only impacts a few legacy customers)